pyqt-apps/siriushla/si_ap_fofb/custom_widgets.py

```diff
@@ -932,6 +936,44 @@ def _setupSYSIDExc(self):
 
         return self._build_scroll_area(wid)
 
+    def _setupErrCntTab(self):
+        wid = QWidget()
+        wid.setToolTip(
+            "If there is any counter increment, "
+            "there is a packet loss problem.")
+        lay = QGridLayout(wid)
+        lay.setSpacing(1)
+        lay.setAlignment(Qt.AlignLeft | Qt.AlignTop)
+
+        # header
+        lay.addWidget(
+            QLabel('<h4>Device</h4>', self, alignment=Qt.AlignCenter), 0, 0)
+        lay.addWidget(
+            QLabel('<h4>Error CH 0</h4>', self, alignment=Qt.AlignCenter),
+            0, 1)
+        lay.addWidget(
+            QLabel('<h4>Error CH 1</h4>', self, alignment=Qt.AlignCenter),
+            0, 2)
+        lay.addWidget(
+            QLabel('<h4>Error CH 2</h4>', self, alignment=Qt.AlignCenter),
+            0, 3)
+        lay.addWidget(
+            QLabel('<h4>Error CH 3</h4>', self, alignment=Qt.AlignCenter),
+            0, 4)
+
+        # table
+        for idx, ctl in enumerate(self.ctrlrs):
+            row = idx + 1
+            lbl = QLabel(ctl, self, alignment=Qt.AlignCenter)
+            lay.addWidget(lbl, row, 0)
+            for i in range(4):
+                pvnerr = _PVName(ctl).substitute(
+                    prefix=self.prefix, propty=f'DCCFMCFrameErrCntCH{i}-Mon')
```

Comment:
@augustofg since we are adding this tab for the FMC counters, do you think it would be reasonable to have another tab for P2P counters?

Reply:
> Hi @ericonr , only for FOFBControllers or for all BPMs?

I think it would make sense to limit to FOFBControllers only for now. The are some crates that the FOFB controller can't see all BPMs directly though and rely on other BPMs to get the beam position data (crates 05 and 06) tough.

Reply:
Hi @ericonr , @augustofg , I included DCCP2P PVs of the FOFB Controllers in the same tab, planning to merge this one and we can think about a better solution in another PR